regression/scripts/AVL/avl_files/aircraft.avl

```diff
@@ -62,7 +62,7 @@ SECTION
 
 SECTION
 #Xle    Yle      Zle      Chord     Ainc  Nspanwise  Sspace
-32.4164746575  1.54    7.777    0.95    0.0     
+32.4164746575  0.0    7.777    0.95    0.0     
```

Comment:
This file seems unrelated.

Reply:
So according to matt...this files is worthless, it is generated everytime the regression is run

Reply:
Make this an issue for AVL coverage